.github/workflows/build-docker-image.yml

```diff
@@ -22,12 +22,12 @@ jobs:
 
     # Setup for buildx
     - name: Set up QEMU
-      uses: docker/setup-qemu-action@v1
-      with: 
+      uses: docker/setup-qemu-action@v2
+      with:
         image: tonistiigi/binfmt:qemu-v6.0.0-12
     - name: Set up Docker Buildx
       id: buildx
-      uses: docker/setup-buildx-action@v1
+      uses: docker/setup-buildx-action@v2
       with:
         version: v0.6.1
```

Comment:
Is this one required? I'm not sure whether dependabot would notify about updates for that version.

Reply:
Version pinning was required for linux/s390x builds, see #26.

Reply:
I've removed the version pinning, because the linux/s390x builds would fail otherwise... although the version pinning had been introduces due to the same error which appears now 🤪